content/spire/try/quickstart.md

```diff
@@ -0,0 +1,209 @@
+---
+title: SPIRE Quickstart
+short: SPIRE Quickstart
+description: How to quickly get SPIRE up and running
+weight: 1
+toc: false
+---
+
+## Overview
+
+In this quick introduction to SPIRE you will learn how to:
+
+* **Start the spire-server process**
+* **Attest a spire-agent to the server using a join token**
+* **Configure a registration entry for a workload**
+* **Fetch an SVID over the WorkloadAPI**
+* **Learn where to find resources for more complex installations on Linux and Kubernetes** 
+
+## Prerequisites
+
+* **A Linux or macOS environment**
+* **The openssl command line tool**
+* **For macOS, Go 1.11 or higher must be installed to build SPIRE. See https://golang.org/dl/ or run `brew install golang`**
+
+The commands in this getting started guide can be run as a standard user or root.
```

Comment:
If it doesn't make a difference, is there value in mentioning this?

Reply:
Mostly to point out that you do _not_ need to be root for this quickstart